client/tracks/index.js

```diff
@@ -60,6 +60,8 @@ const events = {
 	CONNECT_ACCOUNT_LEARN_MORE: 'wcpay_welcome_learn_more',
 	CONNECT_ACCOUNT_STRIPE_CONNECTED: 'wcpay_stripe_connected',
 	CONNECT_ACCOUNT_KYC_MODAL_OPENED: 'wcpay_connect_account_kyc_modal_opened',
+	DEPOSITS_ROW_CLICK: 'wcpay_deposits_row_click',
+	DEPOSITS_DOWNLOAD_CSV_CLICK: 'wcpay_deposits_download',
```

Comment:
I'm using the existing event name. I think it would be nice to change it to make it super clear, e.g. `wcadmin_wcpay_deposits_download_csv_click`. 

I don't think it's in use, so we could rename, but if we want to compare with historical values we'd need to take care.